maud_macros/src/parse.rs

```diff
@@ -155,9 +154,9 @@ impl<'cx, 'a, 'i> Parser<'cx, 'a, 'i> {
             // ???
             _ => {
                 if let [ref tt, ..] = *self.input {
-                    error!(self.cx, tt.span(), "invalid syntax");
+                    self.error(self.cx, tt.span(), "invalid syntax");
```

Comment:
Since `.error()` is a method, it has access to `self` -- so you don't need to pass in `self.cx` separately.